src/lib/apiService.ts

```diff
@@ -91,17 +125,13 @@ const getAccounts = async(currentToken?: string): Promise<FF3Wrapper<FF3Account>
     //   },
     // );
 
-    const response = await http?.get('/accounts', exceptionHandling);
-    if (response && response.status) {
-        if (response.status === 200 && response.data.data) {
-            return response.data.data;
-          } else if (response.status === 401) {
-            return null;
-          }
-    }
 
-    // Return an empty array otherwise
-    return [];
+    const accounts = await Array.fromAsync(pageThroughResource(http, '/accounts')).then((array) => {
```

Comment:
for Array.fromAsync to work, tsconfig.app.json needs to be updated:
`"lib": ["ESNext", "DOM", "DOM.Iterable"],`